datasets/redasa-covid-data.yaml

```diff
@@ -0,0 +1,41 @@
+Name: REDASA COVID-19 Open Data
+Description: >
+  The REaltime DAta Synthesis and Analysis (REDASA) COVID-19 snapshot contains the output of the curation protocol produced by our curator community. A detailed description can be found in [our paper](https://www.jmir.org/2021/5/e25714).
+  The first S3 bucket listed in Resources contains a large collection of medical documents in text format extracted from the [CORD-19 dataset](https://registry.opendata.aws/cord-19/), plus other sources deemed relevant by the REDASA consortium. 
+  The second S3 bucket contains a series of documents surfaced by [Amazon Kendra](https://aws.amazon.com/kendra/) that were considered relevant for each medical question asked. The final S3 bucket contains the GroundTruth annotations created by our curator community.
+Documentation: https://github.com/PanSurg/redasa-sample-data/blob/master/README.md
+Contact: redasa-open-data@imperial.ac.uk
+ManagedBy: REDASA Consortium, Imperial College London, UK
+UpdateFrequency: Yearly updates
+Tags:
+  - aws-pds
+  - COVID-19
+  - coronavirus
+  - life sciences
+  - information retrieval
+  - natural language processing
+  - text analysis
+License: CC-BY-4.0
+Resources:
+  - Description: This is the raw data repository containing a common crawl of CORD-19 papers and other sources identified by the REDASA Project.
+    ARN: arn:aws:s3:::pansurg-curation-raw-open-data
```

Comment:
@oserban currently I cannot list contents for any of these buckets. Did you use our CloudFormation template to generate these? If no, we will want to alter the bucket policy to make these world-readable. We'll also want to make sure you have [bucket logs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-server-access-logging.html) and [CloudWatch usage metrics](https://docs.aws.amazon.com/AmazonS3/latest/userguide/configure-request-metrics-bucket.html) enabled on each bucket.

Reply:
@chueatwork I enabled public access to all the resources, plus the bucket logs and cloudwatch metrics. Please let me know if you can list the contents.